url fixes (#2532)
segmentio/segment-docs: src/connections/destinations/catalog/msg91/index.md

2.  Search for "MSG91" in the Catalog, select it, and choose which of your sources to connect the destination to.

> 3. Enter the "API Key" into your Segment Settings UI which you can find from your [MSG91 dashboard](https://control.msg91.com/signin/) in the API page using the 'API' option in the sidebar. It is recommended that you create a brand new API key for the Segment destination.

Reviewer: ```suggestion
3. Enter the "API Key" into your Segment Settings UI which you can find from your [MSG91 dashboard](https://control.msg91.com/signin/) in the API page using the 'API' option in the sidebar. It's recommended that you create a brand new API key for the Segment destination.
```
